sdk/maps/maps-common/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+- This release is a preview of our efforts to create a client library that is user friendly and
+  idiomatic to the JavaScript ecosystem. The reasons for most of the changes in this update can be found in the
+  [Azure SDK Design Guidelines for TypeScript](https://azure.github.io/azure-sdk/typescript_introduction.html).
```

Comment:
Please update the description here. maybe simply call it "initial release"

Reply:
Updated in f180ee43d86f1b0f0ce3ac88cab08893a91f6671. Also remove the one for `maps-common` following `keyvault-common`